Templates/Multiplayer/Template/Gem/Code/Source/Spawners/RoundRobinSpawner.cpp

```diff
@@ -1,6 +1,6 @@
 /*
```

Comment:
Ideally RoundRobinSpawner.h/cpp and Source/Spawners/IPlayerSpawner.h is deleted entirely, and removed from the cmake file list: https://github.com/o3de/o3de-extras/blob/development/Templates/Multiplayer/Template/Gem/Code/%24%7BNameLower%7D_files.cmake 

Remove game code references to RoundRobinSpawner and IPlayerSpawner as well (I think there's references inside the Source/${Name}SystemComponent.cpp

RoundRobinSpawner is replaced by the built-in Multiplayer gem component called SimplePlayerSpawner.
https://www.docs.o3de.org/docs/user-guide/components/reference/multiplayer/simple-player-spawner/

You'd slap that component to the root level entity (whichever levels Multiplayer template ships with). The component takes the network prefab to spawn, and the player spawn points.

Reply:
Thanks for your input, I created an issue based on that.